src/leappto/cli.py

```diff
@@ -628,8 +634,26 @@ def print_migrate_info(text):
             None
         )
 
-        return_code, claimed_names = mc.check_target()
-        for name in sorted(claimed_names):
+        check_result = dict()
+        return_code, check_result['containers'] = mc.check_target_containers()
+
+        if parsed.status:
+            check_commands = dict()
+            check_commands['docker'] = 'docker info'
+            check_commands['rsync'] = 'rsync --version'
+
+            for service in check_commands.keys():
```

Comment:
I'd suggest `for service, command in check_commands.items()` to save the indirection below.